app/posts/manage-teacher-training-applications/2020-12-18-search-by-candidate-name-or-reference.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Search by candidate name or reference
+description: Let users search the applications list by candidate name or reference
+date: 2020-12-18
+---
+
+We updated the search to allow users to find applications by reference in addition to candidate name.
+
+The search works independently from the filters, allowing users to filter or search in whatever way suits their needs.
+
+This change included:
+
+- moving the search form to above the applications list to accept terms that are compared to a candidate’s full name or reference
+- removing the candidate name search from the filters side bar
+- adding the reference number to each application listing
+- retaining the search term in the search field
+- providing a way to clear the search term once a search has been carried out, allowing users to get back to the original list state
+- updating the text and position of the clear filter action, which also resolved one of the issues highlighted in the recent accessibility review
+
+Other considerations included:
```

Comment:
Maybe put all this in a separate section at the end.

Reply:
Done